lib/components/SelectionControl/radio/radio-button/index.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import clsx from "clsx";
+import {
+    LabelPairedCircleDotMdFillIcon,
+    LabelPairedCircleDotSmFillIcon,
+    LabelPairedCircleInfoMdRegularIcon,
+    LabelPairedCircleInfoSmRegularIcon,
+    LabelPairedCircleMdRegularIcon,
+    LabelPairedCircleSmRegularIcon,
+} from "@deriv/quill-icons";
+import { Text } from "../../../Typography";
+import "./radio-button.scss";
+
+interface IRadio {
```

Comment:
Should it extend `ComponentPrpos<'input'>`?

Reply:
I don't think there's any need for that. Just makes type gymnastics harder without any benefit.